modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,295 @@
+// Module included in the following assemblies:
+//
+// * installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc
+
+:_mod-docs-content-type: Procedure
+[id="installation-osp-local-disk-deployment_{context}"]
+= Deploying {rh-openstack} on local disk
+
+.Prerequisites
+
+* You have an OpenStack cloud with a working Cinder.
+
+* Your OpenStack cloud has at least 75 GB of available storage to accommodate 3 root volumes for the OpenShift control plane.
```

Comment:
75 is the absolute bare minimum. You probably want more, for example [the installer recommends at least 100GB storage per node](https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#machine-pools).

Reply:
Hi @mandre that's why I say "at least 75 GB".

Reply:
As long as it's clear that 25GB root volumes is not suited for production cluster, I'm fine.